docs_app/tools/transforms/templates/api/base.template.html

```diff
@@ -26,6 +26,7 @@ <h1>{$ doc.name $}</h1>
     {% if doc.experimental !== undefined %}<label class="api-status-label experimental">experimental</label>{% endif %}
     {% if doc.stable !== undefined %}<label class="api-status-label stable">stable</label>{% endif %}
     {% if doc.pipeOptions.pure === 'false' %}<label class="api-status-label impure-pipe">impure</label>{% endif %}
```

Comment:
I think I will remove all Angular references like this one - there are so many of them that are so specific to Angular that they could be removed quite safely.